lte/gateway/python/magma/mobilityd/BUILD.bazel

```diff
@@ -20,29 +20,12 @@ DEPS = [
     "//lte/protos:keyval_python_proto",
     "//lte/protos:mconfigs_python_proto",
     "//lte/protos:mobilityd_python_grpc",
-    "//lte/protos:mobilityd_python_proto",
     "//lte/protos:subscriberdb_python_grpc",
-    "//lte/protos:subscriberdb_python_proto",
     "//lte/gateway/python/magma/subscriberdb:sid",
-    "//orc8r/gateway/python/magma/common:metrics",
+    "//orc8r/gateway/python/magma/common:rpc_utils",
```

Comment:
What's happening here? were the deleted deps actually dependencies for rpc_utils?

Reply:
This depends on the deleted dependency. In the case of "common:metrics" it was actually a dependency of "common:service" and thus it is now included in "common:service" as a source file. In other cases the dependency was moved to where it originated.

What we did: 
- Remove all dependencies from mobilityd
- Run `bazel run //...:mobilityd`
- Add missing dependencies at origin of error

Reply:
ah great! That's better. Thanks for the explanation.